modules/0_git/test/main.cpp

```diff
@@ -7,7 +7,7 @@
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
+    EXPECT_EQ(func(a), "Hello, Nizhny!");
```

Comment:
Please start every task by creating a new branch from master:
```
git checkout master
git checkout -b new_task
```
otherwise you will have all the changes from completed modules

Reply:
I created a new one every time. Writing git checkout-b new...

Reply:
or should I first prescribe git checkout master?

Reply:
Yes

Reply:
For now you can just revert main.cpp to origin state - but next just create a branch from master

Reply:
Do I need to fix this in this task or is it said for the following tasks?

Reply:
Yes, please fix it here as well